pkg/discovery/worker/k8s_discovery_worker.go

```diff
@@ -183,8 +195,9 @@ func (worker *k8sDiscoveryWorker) executeTask(currTask *task.Task) *task.TaskRes
 					//glog.Infof("Worker %s finished as expected.", w.GetMonitoringSource())
 					return
 				case <-t.C:
-					glog.Errorf("%s monitoring worker exceeds the max time limit for "+
-						"completing the task.", w.GetMonitoringSource())
+					glog.Errorf("%s monitoring worker from discovery worker %v exceeds the max time limit for "+
+						"completing the task: %v", w.GetMonitoringSource(), worker.id, timeoutSecond)
```

Comment:
Similar as above, would it be helpful to add `resourceMonitorTask` to the log message so that we'll be able to know which node causes timeout when being processed?
Something like:
```
glog.Errorf("%s monitoring worker from discovery worker %v exceeds the max time limit for" + 
"completing the task %s: %v", w.GetMonitoringSource(), worker.id, resourceMonitorTask, timeoutSecond)
```